homeassistant/components/webostv/__init__.py

```diff
@@ -86,8 +87,8 @@ async def async_setup(hass, config):
     """Set up the LG WebOS TV platform."""
     hass.data[DOMAIN] = {}
 
-    async def async_service_handler(service):
-        method = SERVICE_TO_METHOD.get(service.service)
+    async def async_service_handler(service: ServiceCall) -> None:
+        method = SERVICE_TO_METHOD[service.service]
```

Comment:
The service are declared via SERVICE_TO_METHOD, so it can't be absent:
https://github.com/home-assistant/core/blob/6d8c1aa70ce3f23ec22ffdecde0672d726cf7eaf/homeassistant/components/webostv/__init__.py#L96-L100